### PR TITLE
Listing view functionality

### PR DESCRIPTION
**Pull request by amalladi3**
Listing view with description hard coded for now. Kept style almost exactly the same as the table, but stretched and including a contact button. Click on title in the table, routed to this view

### CODE REVIEW

File: src/app/listing-page/listing-page.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     </div>
     <div class="listing-text">
       <div class="listing-name">
-        <a href="#">{{listing.name}}</a>
+        <a href="http://localhost:4200/listing">{{listing.name}}</a>
```
Comment (ScottSWolfe) [inline src/app/listing-page/listing-page.component.html:9]: There's got to be a simple way to do this dynamically, right?

File: src/app/listing-view/listing-view.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import {ListingService} from '../../services/listing.service';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-listing-view',
+  templateUrl: './listing-view.component.html',
+  styleUrls: ['./listing-view.component.css'],
+})
+export class ListingViewComponent implements OnInit {
+  listings: Listing[];
```
Comment (ScottSWolfe) [inline src/app/listing-view/listing-view.component.ts:12]: I don't think `listings` is necessary. If you're only using it to get `listings1`, you could just do `listings1 = res[4]` and remove the instance variable.

File: src/app/listing-view/listing-view.component.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  listing1: Listing;
```
Comment (ScottSWolfe) [inline src/app/listing-view/listing-view.component.ts:13]: I don't like `listing1` as a variable name. Does `listing` not work? If not, maybe we could change the name slightly so we don't need to tack on a number at the end?

File: src/app/listing-page/listing-page.component.html (hunk 1 above)
Comment (amalladi3) [reply src/app/listing-page/listing-page.component.html:9]: yup, working on full stack listing functionality on my local branch and i changed it there

Comment (amalladi3) [reply src/app/listing-page/listing-page.component.html:9]: i'll update this branch soon though

File: src/app/listing-view/listing-view.component.ts (hunk 3 above)
Comment (amalladi3) [reply src/app/listing-view/listing-view.component.ts:13]: yeah forgot to change that, it was there for debugging. wont even need the var with the new code though

File: src/app/listing-view/listing-view.component.ts (hunk 2 above)
Comment (amalladi3) [reply src/app/listing-view/listing-view.component.ts:12]: good point

**Pull request closed without merging**